lldb/test/API/tools/lldb-dap/breakpoint/TestDAP_logpoints.py

```diff
@@ -48,7 +48,9 @@ def test_logmessage_basic(self):
         # 1. First at the loop line with logMessage
         # 2. Second guard breakpoint at a line after loop
         logMessage_prefix = "This is log message for { -- "
-        logMessage = logMessage_prefix + "{i + 3}"
+        message_addr_pattern = r"\b0x[0-9A-Fa-f]+\b"
+        message = '"Hello from main!"'
+        logMessage = logMessage_prefix + "{i + 3}, {message}"
```

Comment:
Does the string literal here need to be an f-string so `message` is properly interpolated?

Reply:
`{message}` is DAP's log message format not python f-string. Changed the test order and variable name to less confusing.